schemas/latest/dbt_yml_files-latest.json

```diff
@@ -209,12 +180,7 @@
       "type": "array",
       "items": {
         "type": "object",
-        "required": [
-          "name",
-          "label",
-          "type",
-          "type_params"
-        ],
+        "required": ["name", "label", "type", "type_params"],
```

Comment:
I assume that these are because your formatting rules are different to what we normally use? They should be able to look at the config in the .vscode settings dir (or whatever it is) to get the style guide